doc/internal/man3/ossl_provider_new.pod

```diff
@@ -171,31 +188,41 @@ Locating a provider module happens as follows:
 
 =item 1.
 
-Look in each directory given by ossl_provider_add_module_location().
+Use the provider object's name as module path, with platform specific
+standard extensions added.
 
 =item 2.
 
-Look in the directory given by the environment variable
-B<OPENSSL_MODULES>.
+If a path was given with ossl_provider_set_module_path(), use that as
+module path instead.
```

Comment:
Move this to the top of the list so they are in priority order.  This would mean losing the comma and beyond.

Reply:
Sounds like the easiest is to merge those two, i.e one sentence saying "uses this if available, otherwise that"

Reply:
I ended up merging the last two items as well.